src/block-components/icon/edit.js

```diff
@@ -109,173 +110,177 @@ export const Edit = props => {
+				max={ 360 }
+				allowReset={ true }
+				placeholder="0"
+				hover={ hover }
+			/>
+
+			{ hasIconPosition && (
+				<AdvancedSelectControl
+					label={ __( 'Icon Position', i18n ) }
+					attribute="iconPosition"
+					options={ [
+						{ value: '', label: __( 'Left', i18n ) },
+						{ value: 'right', label: __( 'Right', i18n ) },
+					] }
 				/>
+			) }
 
+			{ hasIconGap && (
 				<AdvancedRangeControl
-					label={ __( 'Icon Opacity', i18n ) }
-					attribute="iconOpacity"
+					label={ __( 'Icon Gap', i18n ) }
+					attribute="iconGap"
 					min={ 0 }
-					max={ 1 }
-					step={ 0.1 }
+					sliderMax={ 50 }
 					allowReset={ true }
-					placeholder="1.0"
-					hover="all"
+					placeholder="0"
 				/>
+			) }
+		</>
+	)
 
-				<AdvancedRangeControl
-					label={ __( 'Icon Rotation', i18n ) }
-					attribute="iconRotation"
-					min={ 0 }
-					max={ 360 }
-					allowReset={ true }
-					placeholder="0"
-					hover="all"
+	const iconShapeControls = (
+		<>
+			{ filteredShapeColorTypes.length > 1 && (
+				<AdvancedToolbarControl
+					controls={ filteredShapeColorTypes }
+					isSmall={ true }
+					fullwidth={ false }
+					attribute="shapeColorType"
 				/>
+			) }
 
-				{ hasIconPosition && (
-					<AdvancedSelectControl
-						label={ __( 'Icon Position', i18n ) }
-						attribute="iconPosition"
-						options={ [
-							{ value: '', label: __( 'Left', i18n ) },
-							{ value: 'right', label: __( 'Right', i18n ) },
-						] }
-					/>
-				) }
-
-				{ hasIconGap && (
-					<AdvancedRangeControl
-						label={ __( 'Icon Gap', i18n ) }
-						attribute="iconGap"
-						min={ 0 }
-						sliderMax={ 50 }
-						allowReset={ true }
-						placeholder="0"
-					/>
-				) }
-
-			</PanelAdvancedSettings>
-			{ hasShape && (
-				<PanelAdvancedSettings
-					title={ __( 'Icon Shape', i18n ) }
-					id="icon-shape"
-				>
-
-					{ filteredShapeColorTypes.length > 1 && (
-						<AdvancedToolbarControl
-							controls={ filteredShapeColorTypes }
-							isSmall={ true }
-							fullwidth={ false }
-							attribute="shapeColorType"
-						/>
-					) }
+			{ shapeColorControls }
+
+			<AdvancedRangeControl
+				label={ __( 'Shape Border Radius', i18n ) }
+				attribute="shapeBorderRadius"
+				hover="all"
+				min={ 0 }
+				sliderMax={ 100 }
+				step={ 1 }
+				allowReset={ true }
+				placeholder={ 50 }
+			/>
+
+			<AdvancedRangeControl
+				label={ __( 'Shape Padding', i18n ) }
+				attribute="shapePadding"
+				min={ 0 }
+				sliderMax={ 150 }
+				step={ 1 }
+				allowReset={ true }
+				placeholder={ 20 }
+			/>
+
+			<FourRangeControl
+				label={ __( 'Shape Outline Width' ) }
+				units={ [ 'px' ] }
+				min={ 0 }
+				step={ 1 }
+				sliderMax={ 20 }
+				defaultLocked={ true }
+				attribute="shapeOutlineWidth"
+				responsive="all"
+				default="1"
+			/>
+
+			<ColorPaletteControl
+				label={ __( 'Shape Outline Color', i18n ) }
+				attribute="shapeOutlineColor"
+				hasTransparent={ true }
+				hover="all"
```

Comment:
Since `hover` is now an option, this should be:
```suggestion
				hover={ hover }
```